### DISCUSSION TITLE
PPP Bascharage TS vs UNR / GIpsyX Bascharage TS

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#112] [Q&A] [answered]
Hello

As mentioned before we are running PPP for Luxembourg  stations and had some strange results when comparing them to  time series run by other programs.  We have one direction going the opposite way when we compare the graphs. As far as i know i have run the scripts and the setups exactly the same way as previous processing scripts. Have you come across this type of difference before? I do realize that this is a very difficult question and special situation. Any idea were we should look at to find this discrepancy.  We do not think it is RINEX file issue as we ran the same RINEX files with GipsyX as per below graph (and UNR also uses GipsyX for their results).

Thank you for your continued help and feedback

Cheers
Jean  

BASC Position,png - Luxembourg GROOPS script
![Position_basc](https://user-images.githubusercontent.com/100558965/194102003-d177b76a-c013-4603-b999-84bd1f528cfc.png)

Bas4.png - Unversity of Nevada Bascharage TimeSeries
BASCTS.png - Luxembourg GipsyX TimeSeries (2007 - 2014 only - for trend comparison)
![BAS4](https://user-images.githubusercontent.com/100558965/194101249-7f2e4a7e-a638-487a-9fd2-182f8dc5d1e9.png)
![BASCTS](https://user-images.githubusercontent.com/100558965/194101068-8aae2d2a-b5c7-4ceb-bf51-5356c047dfa4.png)

**Comment by tmayerguerr** [comment, marked as answer]
Dear Jean,
I can only guess. But I the processing looks good. Did you forget or do wrong the rotation into the local system (North, East, Up) for plotting (LocalLevelFrame2StarCamera and InstrumentRotate)?
regards
Torsten

**Comment by Bemtgenj** [comment]
Dear Torsten

Txs for the quick reply - so maybe that is the problem as i did not use the locaclLevelFrame2 StarCamera/ Instrument Rotate program in the plot script. this was due to the following problem:   when i (we) tried using those 2 programs with a time series (that was created before doing the  plots) in the plot script i kept on getting errors and so we did not use the rotation portion when we plotted time series. So as it did not work in the plot program i did the rotation portion before creating a time series so that all the elements were are already rotated before creating the final time series (this was to avoid having this problem in the plot program), when i did that i got the following results - which seemed strange and not aligned with the other curves from the other sites.
i guess a simple question would be: if i have a time series from teh processing part how can i plot it using the 2 above programs - do i need to do a special setup in the plot program?
Txs for the help

Cheers
Jean

![Position_basc](https://user-images.githubusercontent.com/100558965/194233619-db7c3ff1-90a0-4bed-b58c-4b40c6ae1e2c.png)

**Comment by tmayerguerr** [comment]
Dear Jean,
I think you compute for every day a new rotation but you have to use only one constant rotation.
In LocallLevelFrame2StarCamera -> constantOriginPerArc=yes
For this the time series must contain only one arc (see InstrumentSynchronize, minGap=0)
regards
Torsten

**Comment by Bemtgenj** [comment]
HI Torsten

Txs for the feedback - much clearer now

Cheers
Jean

**Comment by Bemtgenj** [comment]
Hello

I have run the script for concatenation/ synchronize again (as per above) and then in the plot program in the instrument rotation portion  (i think) i get the following error

**=== Starting GROOPS with 7 processes ===
Config file: <041testPlotsBasc.xml>
--- FileCreateDirectories (/media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc) ---
Create directory </media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc>
Create directory </media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc/tmp>
--- LocalLevelFrame2StarCamera (/media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc/tmp/local2trf_basc.txt) ---
compute local level frame rotation
  4465 of 4465 (time: 00:00:00, remaining: 00:00:00) 
write star camera data to file </media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc/tmp/local2trf_basc.txt>
--- InstrumentRotate (/media/jean/LaCie1/DATA_Linux/groops/spslux/plots/basc/tmp/timeseriespos.basc.txt) ---
read and rotate data </run/user/1001/gvfs/smb-share:server=atlas.uni.lux,share=fstc_geo/geophyresult/spslux/timeseries/basc/resultTS/timeseries_posmean_basc.txt>
     4 of 4465 (time: 00:00:00, remaining: 00:00:00) 
****** Error ******
in process 2 of 7:
in /home/jean/groops/source/config/config.cpp:729 (run)
in /home/jean/groops/source/programs/instruments/instrumentRotate.cpp:115 (run)
in /home/jean/groops/source/parallel/parallel.h:515 (forEachInterval)
rotation for MISCVALUES(3) not implemented
=== Finished GROOPS with error ===
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:**

  Process name: [[3615,1],3]
  Exit code:    1

I am not clear what this error means . I  attached the plot script, log file and the actual time series  that i am using to get the plots plot program, maybe that will help a bit.

Thanks very much for your help.
BR
Jean


[Rotationplot.zip](https://github.com/groops-devs/groops/files/9827245/Rotationplot.zip)

**Reply by tmayerguerr** [reply]
Dear Jean

Within all the messages the error "rotation for MISCVALUES(3) not implemented" is the important information.
As workaround you can change the type of your file with [InstrumentSetType](https://groops-devs.github.io/groops/html/InstrumentSetType.html) to VECTOR3D.

regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

txs for the feedback - i will change the output type and rerun the script.

BR
Jean
